#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-kvers.dpatch by Cyril Brulebois <cyril.brulebois@enst-bretagne.fr>
##
## DP: Replacing uname call with KVERS, overwriting seemed not to work.

@DPATCH@

diff -Naurp open-vm-tools.orig/modules/linux/vmblock/Makefile open-vm-tools/modules/linux/vmblock/Makefile
--- open-vm-tools.orig/modules/linux/vmblock/Makefile	2008-09-08 02:30:11.000000000 +0200
+++ open-vm-tools/modules/linux/vmblock/Makefile	2008-09-13 11:13:08.000000000 +0200
@@ -26,7 +26,7 @@
 ####
 SRCROOT = .
 
-VM_UNAME = $(shell uname -r)
+VM_UNAME = $(KVERS)
 
 # Header directory for the running kernel
 HEADER_DIR = /lib/modules/$(VM_UNAME)/build/include
diff -Naurp open-vm-tools.orig/modules/linux/vmci/Makefile open-vm-tools/modules/linux/vmci/Makefile
--- open-vm-tools.orig/modules/linux/vmci/Makefile	2008-09-08 02:30:11.000000000 +0200
+++ open-vm-tools/modules/linux/vmci/Makefile	2008-09-13 11:13:32.000000000 +0200
@@ -26,7 +26,7 @@
 ####
 SRCROOT = .
 
-VM_UNAME = $(shell uname -r)
+VM_UNAME = $(KVERS)
 
 # Header directory for the running kernel
 HEADER_DIR = /lib/modules/$(VM_UNAME)/build/include
diff -Naurp open-vm-tools.orig/modules/linux/vmhgfs/Makefile open-vm-tools/modules/linux/vmhgfs/Makefile
--- open-vm-tools.orig/modules/linux/vmhgfs/Makefile	2008-09-08 02:30:11.000000000 +0200
+++ open-vm-tools/modules/linux/vmhgfs/Makefile	2008-09-13 11:13:08.000000000 +0200
@@ -26,7 +26,7 @@
 ####
 SRCROOT = .
 
-VM_UNAME = $(shell uname -r)
+VM_UNAME = $(KVERS)
 
 # Header directory for the running kernel
 HEADER_DIR = /lib/modules/$(VM_UNAME)/build/include
diff -Naurp open-vm-tools.orig/modules/linux/vmmemctl/Makefile open-vm-tools/modules/linux/vmmemctl/Makefile
--- open-vm-tools.orig/modules/linux/vmmemctl/Makefile	2008-09-08 02:30:11.000000000 +0200
+++ open-vm-tools/modules/linux/vmmemctl/Makefile	2008-09-13 11:13:08.000000000 +0200
@@ -26,7 +26,7 @@
 ####
 SRCROOT = .
 
-VM_UNAME = $(shell uname -r)
+VM_UNAME = $(KVERS)
 
 # Header directory for the running kernel
 HEADER_DIR = /lib/modules/$(VM_UNAME)/build/include
diff -Naurp open-vm-tools.orig/modules/linux/vmsync/Makefile open-vm-tools/modules/linux/vmsync/Makefile
--- open-vm-tools.orig/modules/linux/vmsync/Makefile	2008-09-08 02:30:11.000000000 +0200
+++ open-vm-tools/modules/linux/vmsync/Makefile	2008-09-13 11:13:08.000000000 +0200
@@ -26,7 +26,7 @@
 ####
 SRCROOT = .
 
-VM_UNAME = $(shell uname -r)
+VM_UNAME = $(KVERS)
 
 # Header directory for the running kernel
 HEADER_DIR = /lib/modules/$(VM_UNAME)/build/include
diff -Naurp open-vm-tools.orig/modules/linux/vmxnet/Makefile open-vm-tools/modules/linux/vmxnet/Makefile
--- open-vm-tools.orig/modules/linux/vmxnet/Makefile	2008-09-08 02:30:11.000000000 +0200
+++ open-vm-tools/modules/linux/vmxnet/Makefile	2008-09-13 11:13:08.000000000 +0200
@@ -26,7 +26,7 @@
 ####
 SRCROOT = .
 
-VM_UNAME = $(shell uname -r)
+VM_UNAME = $(KVERS)
 
 # Header directory for the running kernel
 HEADER_DIR = /lib/modules/$(VM_UNAME)/build/include
diff -Naurp open-vm-tools.orig/modules/linux/vsock/Makefile open-vm-tools/modules/linux/vsock/Makefile
--- open-vm-tools.orig/modules/linux/vsock/Makefile	2008-09-08 02:30:11.000000000 +0200
+++ open-vm-tools/modules/linux/vsock/Makefile	2008-09-13 11:13:43.000000000 +0200
@@ -26,7 +26,7 @@
 ####
 SRCROOT = .
 
-VM_UNAME = $(shell uname -r)
+VM_UNAME = $(KVERS)
 
 # Header directory for the running kernel
 HEADER_DIR = /lib/modules/$(VM_UNAME)/build/include
